RELEASE_PROCESS.md

```diff
@@ -132,10 +127,15 @@ based on the branch you just created.
   change to the live website; however, it might take a few minutes for
   the deploy to complete and then a few more for CDN caches to expire.
 
-- [ ] Publish your draft blog post.
-
-- [ ] Send an email to the chef and chef-dev mailing lists announcing
-the release with a link to the blog post.
+- [ ] Write and then publish a Discourse post on https://discourse.chef.io
+  once the release is live. This post should contain a link to the downloads
+  page ([https://downloads.chef.io](https://downloads.chef.io)) and its contents
+  should be based on the information that was added to the RELEASE_NOTES.md file
+  in an earlier step. *The post should  be published to the Chef Release
+  Announcements category on https://discourse.chef.io/. If it is a security
+  release, it should also be published to the Chef Security Announcements
+  category.* Full details on the policy of making release announcements on
+  Discourse can be found on the wiki: [https://chefio.atlassian.net/wiki/display/ENG/Release+Announcements+and+Security+Alerts](https://chefio.atlassian.net/wiki/display/ENG/Release+Announcements+and+Security+Alerts)
```

Comment:
If we leave it, I'd say like `Chef internal wiki:` so there's no question you're not supposed to have a login.